http/src/main/java/io/hyperfoil/http/api/HttpRequest.java

```diff
@@ -24,11 +24,14 @@ public class HttpRequest extends Request {
    public HttpMethod method;
    public String authority;
    public String path;
-   public CacheControl cacheControl = new CacheControl();
+   private final boolean httpCacheEnabled;
```

Comment:
Overall the instance size is 88 bytes, with or without this boolean.. due to some `alignment/padding gap` we reach `88` in both cases (at least based on the JOL).

On the other hand, I agree that there is no particular reason in keeping this boolean as it is just kept to avoid `cacheControl != null`. I am expecting the difference between the two checks is in terms of nanoseconds, so I think it could be considered a negligibile, right?